pkgs/tools/text/html-tidy/default.nix

```diff
@@ -11,7 +11,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ZMz0NySxzX2XHiqB8f5asvwjIG6kdIcq8Gb3EbAxBaU=";
   };
 
-  nativeBuildInputs = [ cmake libxslt/*manpage*/ ];
+  # https://github.com/htacg/tidy-html5/pull/1036
```

Comment:
Heh, why not fetchpatch the PR contents and use that?

Reply:
(unconditionally!)

Reply:
Done